app/src/main/java/com/github/se/eduverse/ui/others/setting/Setting.kt

```diff
@@ -1,116 +1,238 @@
+                    modifier = Modifier.fillMaxWidth().testTag("addAccountButton"),
+                    colors = ButtonDefaults.buttonColors(containerColor = SecondaryColor)) {
+                      Icon(
+                          imageVector = Icons.Default.PersonAdd,
+                          contentDescription = "Add Account",
+                          modifier = Modifier.padding(end = 8.dp),
+                          tint = Color.White)
+                      Text(text = "Add Account", color = Color.White)
                     }
-                  }
+                Spacer(modifier = Modifier.height(8.dp))
+                Button(
+                    onClick = { logout(navigationActions) },
+                    modifier = Modifier.fillMaxWidth().testTag("logoutButton"),
+                    colors = ButtonDefaults.buttonColors(containerColor = Color(0xFFD4DEE8))) {
+                      Icon(
+                          imageVector = Icons.Default.ExitToApp,
+                          contentDescription = "Log Out",
+                          modifier = Modifier.padding(end = 8.dp),
+                          tint = Color.Black)
+                      Text(text = "Log Out", color = Color.Black)
+                    }
+              }
             }
+      }
+}
 
-        Spacer(modifier = Modifier.height(16.dp))
+// Other helper functions (SettingsOption and SettingsDropdown) remain the same
 
-        Row(
-            modifier = Modifier.fillMaxWidth().testTag("settingsButtons"),
-            horizontalArrangement = Arrangement.SpaceEvenly) {
-              Button(
-                  onClick = { navigationActions.goBack() },
-                  modifier = Modifier.weight(1f).testTag("saveButton")) {
-                    Text(text = "Save")
-                  }
-              Spacer(modifier = Modifier.width(8.dp))
-              Button(
-                  onClick = { navigationActions.goBack() },
-                  modifier = Modifier.weight(1f).testTag("cancelButton")) {
-                    Text(text = "Cancel")
-                  }
-            }
+// Other helper functions (SettingsOption and SettingsDropdown) remain the same
+
+@Composable
+fun SettingsOption(
+    title: String,
+    icon: ImageVector,
+    navigationActions: NavigationActions,
+    route: String
+) {
+  Row(
+      modifier =
+          Modifier.fillMaxWidth()
+              .clickable { navigationActions.navigateTo(route) }
+              .padding(16.dp)
+              .testTag("settingsOption_$title"),
+      verticalAlignment = Alignment.CenterVertically) {
+        Icon(
+            imageVector = icon,
+            contentDescription = null,
+            tint = SecondaryColor,
+            modifier = Modifier.padding(end = 16.dp))
+        Text(title, modifier = Modifier.weight(1f), color = SecondaryColor)
+        Icon(
+            imageVector = Icons.Default.ArrowForward, contentDescription = null, tint = HeaderColor)
       }
```

Comment:
Good idea to avoid repetition. 
I think you should create empty screens for notifications, saved and archive screens. This way, when said screens will be implemented we won't have to update this one. Also it will make the temporary version better, as we will be able to see how the navigation works instead of the app crashing when those buttons are pressed.